config/workflows/stale.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           days-before-stale: 28
           days-before-close: 14
           days-before-pr-close: -1
-          only-labels: 'bug,needs info/discussion'
+          only-labels: 'bug,needs info'
           exempt-issue-labels: 'no-stale'
           stale-issue-message: >-
```

Comment:
On https://github.com/nextcloud/android/blob/c8b1e0e43d404c4bb1e3237946cba1c019dcf79c/.github/workflows/stale.yml#L17

we have this without "> -" 
Not sure how this behaves.`?

Reply:
I changed that on purpose, they're modifiers for the multiline string. Check https://yaml-multiline.info/ for interactive explanation, but in summary, `>` changes newlines for spaces, and `-` removes the last newline.

In the end it should result in the message being a single line, and not manually broken in three lines with the quotes.